### ISSUE TITLE
boost::filesystem::space() does not throw an exception under Windows anymore

### ISSUE THREAD

**Issue by pkl97** [boostorg/filesystem#167]
In Boost 1.71.0 the program below threw an exception under Windows and Linux due to the non existing path.
In Boost 1.74.0 the exception under Linux is still there but under Windows the program returns 0.

```
#include <iostream>
#include <boost/filesystem.hpp>

int main()
{
    try
    {
        boost::filesystem::space("DoesNotExist");
        return 0;
    }
    catch (const std::exception& e) {
        std::cout << e.what() << std::endl;
    }
    return 1;
}
```

**Comment by pkl97** [comment]
@Lastique Could this change be integrated into the upcoming Boost 1.75 release?

**Comment by Lastique** [comment]
I will try to include it.

**Comment by Lastique** [comment]
Merged to master, should be released with 1.75.

**Comment by pkl97** [comment]
Great, thanks a lot!
